aldryn_faq/admin.py

```diff
@@ -21,7 +21,9 @@
 from .forms import CategoryAdminForm
 
 
-class CategoryAdmin(AllTranslationsMixin, TranslatableAdmin):
+class CategoryAdmin(VersionedPlaceholderAdminMixin,
+                    AllTranslationsMixin,
```

Comment:
Minor thing and may have no material impact, but, I'd probably make `AllTranslationsMixin` the left-most one.